packages/atlas/src/components/_crt/CloseMarketModal/CloseMarketModal.tsx

```diff
@@ -15,26 +22,58 @@ const getTokenDetails = (id: string) => ({
 })
 
 export type CloseMarketModalProps = {
-  tokenId: string
+  channelId: string
+  show: boolean
+  onClose: () => void
 }
 
-export const CloseMarketModal = ({ tokenId }: CloseMarketModalProps) => {
-  const { title, pricePerUnit, userCrtToken, tokenRequiredToSell } = getTokenDetails(tokenId)
+export const CloseMarketModal = ({ onClose, show, channelId }: CloseMarketModalProps) => {
+  const { title, pricePerUnit, userCrtToken, tokenRequiredToSell } = getTokenDetails('1')
+  const { joystream, proxyCallback } = useJoystream()
+  const { displaySnackbar } = useSnackbar()
+  const { memberId } = useUser()
+  const handleTransaction = useTransaction()
+  const client = useApolloClient()
 
   const hasInsufficientTokens = userCrtToken < tokenRequiredToSell
 
+  const handleCloseAmm = useCallback(async () => {
+    if (!joystream || !channelId || !memberId) {
+      return
```

Comment:
probably we should log in sentry that something went wrong here and user was unable to close the amm